docker/dev/Dockerfile

```diff
@@ -89,9 +89,11 @@ RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" \
 
 # install debugger
 RUN \
+      if [ "$GOLANG_OS_ARCH" == "linux-amd64" ]; \
+      then \
 	go get -u github.com/go-delve/delve/cmd/dlv && dlv version; \
```

Comment:
This is the command which installs the debugger which I believe will work for ARM.